common/scripts/lib/_helpers.sh

```diff
@@ -34,6 +34,11 @@ __check_os_and_architecture() {
     # it as "Ubuntu" so that will not change here.
     local os=$(echo "$NAME" | cut -f 1 -d ' ')
     local os_version="$VERSION_ID"
+    if [ "$os" == "Red" ]; then
```

Comment:
Compare the whole string `NAME` and if its "Red Hat..." set to RHEL and skip `os` parsing.